packages/dev/core/src/Engines/Native/nativeInterfaces.ts

```diff
@@ -99,10 +99,15 @@ export interface INativeEngine {
     submitCommands(): void;
 }
 
+/** @internal */
+interface INativeEngineFeatures {
+    nonFloatVertexBuffers: boolean;

```

Comment:
If this flag exists on the features object, it will be true.
```suggestion
    nonFloatVertexBuffers: true;

```

Reply:
If it is always true why should we pass it in from the JS to native ?

Reply:
It will sometimes be missing.